app/components/pipeline/Header/index.js

```diff
@@ -189,7 +174,7 @@ class Header extends React.Component {
     if (this.props.pipeline.description) {
       return <Emojify text={this.props.pipeline.description} />;
     } else {
-      return this.repositoryUrl();
+      return this.props.pipeline.repository.url;
```

Comment:
The `provider.url` will always be a web URL tho… if you check out the `Project::Provider::*` classes you can see they each have their own `project_url` methods, which is what we've been using until now to link to commits, pull requests, etc.